Jenkinsfile

```diff
@@ -870,22 +870,30 @@ pass = ${OBS_CREDENTIALS_PASSWORD}
             }
             // Save logs of failed builds to s3 - we want to analyze where we may have jenkins issues.
             script {
-                if ((env.BRANCH_NAME == 'develop' || env.BRANCH_NAME == 'master') && fileExists('bin/clean-jenkins-log')) {
+                if (env.BRANCH_NAME == 'develop' || env.BRANCH_NAME == 'master') {
                     writeFile(file: 'build.log', text: getBuildLog())
-                    sh 'bin/clean-jenkins-log'
+                    sh """ 
+                         bin/clean-jenkins-log
+                         container-host-files/opt/scf/bin/klog.sh -f ${jobBaseName()}-${BUILD_NUMBER}-scf
+                       """
                     withAWS(region: params.S3_REGION) {
                         withCredentials([usernamePassword(
                             credentialsId: params.S3_CREDENTIALS,
                             usernameVariable: 'AWS_ACCESS_KEY_ID',
                             passwordVariable: 'AWS_SECRET_ACCESS_KEY',
                         )]) {
                             script {
-                                def subdir = "${params.S3_PREFIX}${distSubDir()}"
                                 def prefix = distPrefix()
+                                def subdir = "${params.S3_PREFIX}${distSubDir()}${prefix}${env.BUILD_TAG}/"
```

Comment:
I think this is the one generating weird directory names like `PR-2170-jenkins-scf-PR-2170-6`. No need for `PR-2170` showing up twice.

Reply:
This one even seemed bit odd to me, however I kept the naming convention consistent with what was already there.

One option is to get rid of prefix which is essentially:`PR-${CHANGE_ID}-` and keep `env.BUILD_TAG`. Though this convention will diverge from rest of the script where prefix is being used

Reply:
I am ok with this.

Reply:
@andreas-kupries just to be clear, are you okay with the existing convention or my proposal?

As I haven't implemented any changes from the proposal yet for this.

Reply:
At least add a slash after `${prefix}` to make it a folder. We don't want to mess with the BUILD_TAG right now.

Reply:
I am ok with the existing convention